pkg/pxc/app/statefulset/node.go

```diff
@@ -210,22 +221,28 @@ func (c *Node) PMMContainer(spec *api.PMMSpec, secrets string, v120OrGreater boo
 		},
 	}
 
-	return ct
+	return ct, nil
 }
 
 func (c *Node) Resources(spec *api.PodResources) (corev1.ResourceRequirements, error) {
 	return app.CreateResources(spec)
 }
 
-func (c *Node) Volumes(podSpec *api.PodSpec) *api.Volume {
+func (c *Node) Volumes(podSpec *api.PodSpec, version130Compare int) *api.Volume {
 	vol := app.Volumes(podSpec, DataVolumeName)
 	ls := c.Labels()
 	vol.Volumes = append(
 		vol.Volumes,
 		app.GetTmpVolume(),
 		app.GetConfigVolumes("config", ls["app.kubernetes.io/instance"]+"-"+ls["app.kubernetes.io/component"]),
+		app.GetConfigVolumes("auto-config", "auto-"+ls["app.kubernetes.io/instance"]+"-"+ls["app.kubernetes.io/component"]),
 		app.GetSecretVolumes("ssl-internal", podSpec.SSLInternalSecretName, true),
 		app.GetSecretVolumes("ssl", podSpec.SSLSecretName, podSpec.AllowUnsafeConfig))
+	if version130Compare > 0 {
```

Comment:
what if we change in 1.4.0 volume logic again? are we going to add one more integer values to this function?